pom.xml

```diff
@@ -47,43 +47,11 @@
     </license>
   </licenses>
 
-  <repositories>
```

Comment:
🐛 In such case it won't be able to retrieve Parent pom if it is missing in the local cache.

OTOH you can simplify it to
```xml
    <repositories> 
      <repository>
        <id>repo.jenkins-ci.org</id>
        <url>http://repo.jenkins-ci.org/public/</url>
      </repository>
    </repositories>
    <pluginRepositories>
      <pluginRepository>
        <id>repo.jenkins-ci.org</id>
        <url>http://repo.jenkins-ci.org/public/</url>
      </pluginRepository>
    </pluginRepositories>
```